versioned_docs/version-v7/developing/config.md

```diff
@@ -178,6 +178,7 @@ Below are the config options that Ionic uses.
 | `loadingEnter`              | `AnimationBuilder`                                                                | Provides a custom enter animation for all `ion-loading`, overriding the default "animation".                                                                                                                                                                                        |
 | `loadingLeave`              | `AnimationBuilder`                                                                | Provides a custom leave animation for all `ion-loading`, overriding the default "animation".                                                                                                                                                                                        |
 | `loadingSpinner`            | `SpinnerTypes`                                                                    | Overrides the default spinner for all `ion-loading` overlays.                                                                                                                                                                                                                       |
+| `logLevel`                  | `'OFF' \| 'ERROR' \| 'WARN'`                                                      | Configures the logging level for Ionic Framework. If `'OFF'`, no errors or warnings are logged. If `'ERROR'`, only errors are logged. If `'WARN'`, errors and warnings are logged.                                                                                                  |
```

Comment:
Since the `logLevel` is only being introduced to version 8, we don't need to include it in the v7 docs.

Reply:
Ahhh good catch 😂

Reply:
https://github.com/ionic-team/ionic-docs/pull/3998/commits/94470b0cf3ef00a8a984baa80f5b0248d9f82a7d